game_eggs/among_us/impostor_server/egg-among-us--impostor-server.json

```diff
@@ -1,17 +1,18 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1",
+        "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2021-07-23T12:00:05+03:00",
+    "exported_at": "2023-01-18T15:43:34+01:00",
     "name": "Among Us - Impostor Server",
     "author": "info@goover.de",
-    "description": "Impostor is one of the first Among Us private servers, written in C#.\r\n\r\nThe latest version supported is 2022.4.19, both desktop and mobile.\r\n\r\nThere are no special features at this moment, the goal is aiming to be as close as possible to the real server, for now. In a later stage, making modifications to game logic by modifying GameData packets can be looked at.",
+    "description": "Impostor is one of the first Among Us private servers, written in C#.\r\n\r\nThere are no special features at this moment, the goal is aiming to be as close as possible to the real server, for now. In a later stage, making modifications to game logic by modifying GameData packets can be looked at.",
     "features": null,
-    "images": [
-        "ghcr.io/parkervcp/yolks:dotnet_6"
-    ],
+    "docker_images": {
+        "Dotnet_7": "ghcr.io\/parkervcp\/yolks:dotnet_7",
+        "Dotnet_6": "ghcr.io\/parkervcp\/yolks:dotnet_6"
```

Comment:
can be removed, becasue only 7.0 is supported

Reply:
Its fine. 1 version older still uses dotnet 6. And dotnet 7 will be selected by default